Tests/ComposableArchitectureTests/TimerTests.swift

```diff
@@ -126,7 +126,7 @@ final class TimerTests: XCTestCase {
     scheduler.advance(by: 1)
     XCTAssertEqual(count, 3)
 
-    scheduler.run()
-    XCTAssertEqual(count, 3)
+//    scheduler.run()
+//    XCTAssertEqual(count, 3)
```

Comment:
prob wanna bring these back, hope they pass!

```suggestion
    scheduler.run()
    XCTAssertEqual(count, 3)
```